disnake/shard.py

```diff
@@ -27,10 +27,24 @@
 
 import asyncio
 import logging
-from typing import TYPE_CHECKING, Any, Callable, Dict, List, Optional, Tuple, Type, TypeVar
+from typing import (
+    TYPE_CHECKING,
+    Any,
+    Callable,
+    Dict,
+    List,
+    Optional,
+    Tuple,
+    Type,
+    TypeVar,
+    Union,
+    overload,
+)
 
 import aiohttp
 
+from disnake.i18n import LocalizationProtocol
```

Comment:
this could be a type-checking-only import + relative as well

Reply:
.

Reply:
Fixed in 1a642234474e2777ce36aa5d754ca488f2cd2024.